src/shogun/classifier/svm/WDSVMOcas.cpp

```diff
@@ -60,8 +60,8 @@ CWDSVMOcas::CWDSVMOcas()
 }
 
 CWDSVMOcas::CWDSVMOcas(E_SVM_TYPE type)
-: CMachine(), use_bias(false), bufsize(3000), C1(1), C2(1),
-	epsilon(1e-3), method(type)
+    : CStringFeaturesDispatch<CWDSVMOcas, CMachine>(), use_bias(false),
```

Comment:
we would need an integration (or unit) test for this class with string data before we can merge this.
Could you send an example in a separate PR?
@vinx13 what is the state of string feature meta examples?

Reply:
with string feature factory, we can port meta examples now

Reply:
@vinx13 would you mind adding an example for the `CWDSVMOcas` ? :) (no cookbook needed for now)

Reply:
allright